timesketch/lib/analyzers/sequence_sessionizer.py

```diff
@@ -0,0 +1,158 @@
+"""Sequenced activity sessionizing sketch analyzer plugin."""
+
+from __future__ import unicode_literals
+
+from timesketch.lib.analyzers import sessionizer
+
+
+class SequenceSessionizerSketchPlugin(sessionizer.SessionizerSketchPlugin):
+    """Base class for specialized sequenced activity sessionizings
+    sketch plugins.
+
+    Attributes:
+        event_seq: List of dictionaries of attributes describing the events.
+        event_storage: List of events.
+        num_event_to_find: Number that shows which event from the sequence
+            should be matched.
+        recording: Shows if currently are finded and stored events that match
+            the specified sequence of events.
+        return_fields: List of name of event attributes, should be specified in
+            the inheriting sessionizers. It must contains 'timestamp'.
+        session_num: Counter for the number of sessions.
+        session_type: The name of the event sequence.
+    """
+    event_seq = None
+    event_storage = []
+    num_event_to_find = 0
+    recording = False
+    return_fields = ['timestamp']
+    session_num = 0
+    session_type = None
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Allocates each event between the first event of the event_seq and the
+        last event of the event_seq an session_type attribute and a session_num.
+
+        Returns:
+            String containing the name of the event sequence and the
+            number of sessions created.
+        """
+        if self.session_type is None:
+            raise RuntimeError('No session_type provided.')
+        if self.event_seq is None or self.event_seq == []:
+            raise RuntimeError('No event_seq provided.')
+        # If return_fields in none, then all attributes are provided.
+        if self.return_fields is not None:
+            if 'timestamp' not in self.return_fields:
+                raise RuntimeError('No "timestamp" in return_fields.')
+            if len(self.return_fields) <= 1:
+                raise RuntimeError('No return_fields provided.')
```

Comment:
Either change the condition to `<= 0` or change the error message to `"Not enough return_fields provided"`